bundles/org.openhab.binding.octopusenergy/README.md

```diff
@@ -0,0 +1,167 @@
+# Octopus Energy Binding
+
+This binding allows OpenHAB to communicate with the public API from Octopus Energy (https://octopus.energy), an energy provider in the UK.
+The following features are provided by the binding:
+
+- retrieval of latest meter readings (consumption)
+- retrieval of tariff schedule / unit rates (if on an agile tariff, half-hourly future rates are provided based on availability)
+- calculation of forward looking least cost window based on given energy consumption. I.e. when is the cheapest time to run my dishwasher? This is available as a single pair of channels (cheapestSlotDuration, cheapestSlotStart) and also as an action for the use in scripts/rules.
+
+#### Current Limitations / Work in Progress
+
+- only tested with Octopus "Agile" electricity tariff. I would welcome users of "Go" or "Go Faster" for testing.
```

Comment:
Start sentences with a capital letter. Also what do you think of removing the heading and just simply stating that "Only tested with Octopus "Agile" electricity tariff." ?

Reply:
done